### PR TITLE
Fix some python bindings issues

### PR DESCRIPTION
**Pull request by akva2**
Issues building with lto
- missing source file
- some libfmt issues (mostly also seen with lto)

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: python/setup.py.in [hunk 1]
```diff
@@ -60,13 +60,14 @@ ext_modules = [
                 'cxx/summary_state.cpp',
                 'cxx/table_manager.cpp',
                 'cxx/well.cpp',
-                'cxx/export.cpp'
+                'cxx/export.cpp',
+                'cxx/builtin_pybind11.cpp'
```
Comment (blattms) [inline python/setup.py.in:64]: It is probably me, but I cannot find this file (either here or in master)-

Comment (bska) [reply python/setup.py.in:64]: It (`cxx/builtin_pybind11.cpp`) is [automatically generated](https://github.com/OPM/opm-common/blob/877f82ef06bf186be0d6e1fd902af032fb01495b/src/opm/parser/eclipse/Parser/createDefaultKeywordList.cpp#L65-L66) if you [enable Python support](https://github.com/OPM/opm-common/blob/877f82ef06bf186be0d6e1fd902af032fb01495b/GenerateKeywords.cmake#L113-L117).

Comment (blattms) [reply python/setup.py.in:64]: Thanks. looks good to me then.

**Pull request merged**